mne_sandbox/preprocessing/dss.py

```diff
@@ -0,0 +1,108 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Tue Apr 19 12:47:10 2016
+@author: drmccloy
+"""
+
+import numpy as np
+
+
+def dss(epochs, data_n_keep=None, bias_n_keep=None, data_thresh=None,
+        bias_thresh=None):
```

Comment:
You probably don't want to allow `None` thresholding because you can end up with zero-value eigenvalues, right? Instead you could do e.g. `data_thresh='numerical'` which would use numerical precision, similar to `scipy.linalg.orth` [here] (https://github.com/scipy/scipy/blob/v0.14.0/scipy/linalg/decomp_svd.py#L181). If a user really wants to potentially break things they can go ahead and set it to zero themselves.